client/my-sites/email/email-providers-comparison/price-badge/index.tsx

```diff
@@ -4,18 +4,15 @@ import type { ReactElement } from 'react';
 import './style.scss';
 
 type PriceBadgeProps = {
-	additionalPriceInformationComponent?: ReactElement | null;
-	priceComponent: ReactElement;
+	additionalPriceInformation?: ReactElement | null;
+	price: ReactElement;
 };
 
-const PriceBadge = ( {
-	additionalPriceInformationComponent,
-	priceComponent,
-}: PriceBadgeProps ): ReactElement => (
+const PriceBadge = ( { additionalPriceInformation, price }: PriceBadgeProps ): ReactElement => (
 	<div className="price-badge">
 		<PromoCardPrice
-			formattedPrice={ priceComponent }
-			additionalPriceInformation={ <span>{ additionalPriceInformationComponent }</span> }
+			formattedPrice={ price }
+			additionalPriceInformation={ <span>{ additionalPriceInformation }</span> }
```

Comment:
This is just a minor refactoring - there is no need to have those suffixes for those particular components.